distribution/pom.xml

```diff
@@ -1377,6 +1377,18 @@
             <version>${project.version}</version>
             <type>jar</type>
         </dependency>
+	<dependency>
+		<groupId>org.openhab.io</groupId>
+		<artifactId>org.openhab.io.caldav</artifactId>
+		<version>${project.version}</version>
+		<type>jar</type>
+	</dependency>
```

Comment:
That was some sort of merge problem, not sure why merging upstream/master brought that in. Wil check it out.